editoast/src/schema/operational_point.rs

```diff
@@ -22,6 +23,27 @@ pub struct OperationalPoint {
     pub extensions: OperationalPointExtensions,
 }
 
+impl OperationalPoint {
+    
+    pub fn persist(operational_points: &Vec<OperationalPoint>, infrastructure_id: i32, conn: &PgConnection) -> Result<(), Box<dyn ApiError>> {
```

Comment:
Could be renamed `persist_batch`

Reply:
It could be used out the batch import scope. What about persist_all instead ?

Reply:
What do you mean by "It could be used out the batch import scope" ?
This function save operational points as batch.

Reply:
I mean "batch" is not a good term IMO here. This method could be used in other use cases, that do not relies on batch purpose. I guess I don't like the "technical" term. I'd prefer persist_all or something more functional. If you do not agree, this is not really important for me so I can rename

Reply:
> What do you mean by "It could be used out the batch import scope" ? This function save operational points as batch.

I understand what you mean, but we do not care about how it persists here IMO, from the caller perspective at least. The method signature is self sufficient for understand what it does.

Reply:
I applied your advice anyway